src/main/java/org/owasp/esapi/logging/appender/LogPrefixAppender.java

```diff
@@ -51,11 +53,32 @@ public LogPrefixAppender(boolean logUserInfo, boolean logClientInfo, boolean log
         this.logServerIp = logServerIp;
         this.logApplicationName = logApplicationName;
         this.appName = appName;
+        this.logPrefix = true;
```

Comment:
I'd be in favor of removing this constructor entirely. That can have impacts if anyone has actually extended the class, so I suspect that we would not want to do that.

Assuming that we cannot delete this code, I'd like to request two more adjustments here:
1.  Update the Javadoc to specify the logPrefix value is configured to true
2. Chain the constructors - have this one call the other
```
public LogPrefixAppender(boolean logUserInfo, boolean logClientInfo, boolean logServerIp, boolean logApplicationName, String appName) {
  this (logUserInfo, logClientInfo, logServerIp, logApplicationName, appName, true);
}
```